### PR TITLE
A bunch of little updates & additions.

### PR DESCRIPTION
**Pull request by GabuTheDev**
###### Hope it covers everything that I've changed...

| update                                        | comment                                                                 |
| --------------------------------------------- | ----------------------------------------------------------------------- |
| Fixed table of contents.                      | *chatbots redirect was wrong.*                                          |
| Added osu!stream                              | *under the official osu! projects category.*                            |
| Moved Danser!                                 | *from `Alternative game clients` to `Streaming tools`*                  |
| Updated a deprecated beatmap mirror           | *`Kitsu` is now `osu.direct`*                                           |
| Added new heading for the chatbots category.  | *was mentioned in the table of contents but didn't exist.*              |
| Renamed chatbot heading `1`                   | *From `Chatbots (Discord)` to `Discord chatbots`.*                      |
| Fixed typos in discord bot's descriptions `1` | *`recoomendations` is not the right spelling.*                          |
| Renamed chatbot heading `2`                   | *From `Chatbots (in-game)` to `In-Game chatbots`.*                      |
| Fixed typos in discord bot's descriptions `2` | *I said, `recoomendations` is not the right spelling.*                  |
| New in-game chatbot                           | *Added `Elitebotix`*                                                    |
| Fixed typos in discord bot's descriptions `3` | *Nope, `reccomendations` is not a word.*                                |
| Added new heading for lobby bots.             | *above that, I also added 3 bots in there..*                            |
| Added new streaming tool                      | *The well-known `gosumemory`.*                                          |
| Here's Danser!                                | *from `Alternative game clients` to `Streaming tools`*                  |
| Changed style of warning (osuplus)            | *`requires osu! APIv1 key` is now more subtle.*                         |
| Changed name of script                        | *From `osu-web extended` to `osu!web enhanced`*                         |
| Added new userscript                          | *Cpol's website color changer*                                          |
| Added 2 new items in `Collections`            | *Those being: `osucollector!` & `osu-pps`*                              |
| Added new item in "Mapping"                   | *The `Mapset Verifier`*                                                 |
| Fixed name in "Mapping" `1`                   | *`mapping_tools` -> `Mapping Tools`*                                    |
| Fixed name in "Mapping" `2`                   | *`bezier=approximation` -> `bezier-approximation`*                      |
| Fixed name in "Mapping" `3`                   | *`Image-to-Anchors` -> `Image to Anchors`*                              |
| Changed description for `osuskinner`          | *now:  `Online tool for skin mixing and creation.`*                     |
| Added 2 new items in `Skin listing`           | *The well-known `Skinship Compendium` & `Osuck Skins`*                  |
| Created new heading called `Skin tools`       | *Together with 2 new items: `OsuSkinMixer` & `OsuSkinChecker`*          |
| Changed description for `ossapi`              | *now: `A definitive python wrapper for the osu! api.`*                  |
| Added description for `osu-api-extended`      | *now: `Package for advanced work with "osu" api`*                       |
| Moved `rosu` above `rosu-v2`                  | *'cause it looks better idk ¯\\\_(ツ)_/¯*                               |
| Added description for `rosu`                  | *now: `A rust wrapper for osu!apiv1.`*                                  |
| Added 5 new items to `Other projects`         | *`osumapper` - `Osekai.net` - `Rewind` -  `CircleGuard` - `gosumemory`* |
| Renamed item from `Other projects`            | *`osu-trainer` -> `FunOrange's osu!trainer`*                            |

### CODE REVIEW

**Comment by NiceAesth** [comment]
this is insanely opinionated, not happy with this

**Comment by GabuTheDev** [comment]
> this is insanely opinionated, not happy with this

I'm sorry to hear that. I can revert some changes if that's any helpful.
Most of the renames were made to match the Github name/desc & to match the looks of others from the page.
If you're are unhappy with any of the links I've added, I can easily remove them.

**Comment by NiceAesth** [comment]
will go over everything and do a review sometime later

**Comment by NiceAesth** [comment]
@jacksonisiah would like your input

File: README.md [hunk 1]
```diff
@@ -91,37 +93,50 @@ Neither of these are official, however many users have reported success with the
 | [MissAnalyzer](https://top.gg/bot/752035690237394944) | osu! bot for replay analysis                                          |
 | [Yuna](https://top.gg/bot/832597585923014676)         | osu! replay recording bot                                             |
 | [Shisha](https://shisha.mezo.xyz/)                    | osu! replay recording bot approved by the creator of [danser][danser] |
-| [Sombrax79](https://ost.sombrax79.org/commands)       | Bot for stamina training recoomendations                              |
+| [Sombrax79](https://ost.sombrax79.org/commands)       | Bot for stamina training recommendations                              |
 
-### Bots (in-game)
+### In-Game chatbots
```
Comment (jacksonisiah) [inline README.md:98]: Not sure why this was changed

File: README.md [hunk 2]
```diff
@@ -37,6 +38,7 @@ An awesome-style list of osu! projects built by the community. Circle People is
 | ----------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
 | [osu!](https://osu.ppy.sh)                            | The official website for the rhythm game osu!.                                                       |
 | [osu!lazer](https://github.com/ppy/osu)               | osu!lazer source code, the next generation client for osu!                                           |
+| [osu!stream](https://osustream.com/)                  | A very early adaptation of osu! for mobile devices, not maintained.                                  |
```
Comment (jacksonisiah) [inline README.md]: Maybe
```suggestion
| [osu!stream](https://osustream.com/)                  | An official port of osu! on iOS and Android devices from a past era.                                  |
```

File: README.md [hunk 3]
```diff
@@ -142,21 +158,34 @@ Neither of these are official, however many users have reported success with the
...
-| URL                                   | Description             |
-| ------------------------------------- | ----------------------- |
-| [osuskinner](https://osuskinner.com/) | Skin generator for osu! |
+| URL                                   | Description                               |
+| ------------------------------------- | ----------------------------------------- |
+| [osuskinner](https://osuskinner.com/) | Online tool for skin mixing and creation. |
 
 ### Skin listing
 
 ![wip](https://img.shields.io/badge/note-Work%20in%20progress-yellow)
```
Comment (jacksonisiah) [inline README.md]:
```suggestion

```

File: README.md [hunk 4]
```diff
@@ -169,10 +198,10 @@ Neither of these are official, however many users have reported success with the
...
 | [aiosu](https://github.com/niceaesth/aiosu)                       | python     | Async Python library for everything osu! related                                      |
-| [ossapi](https://github.com/circleguard/ossapi)                   | python     | Python library for interacting with the osu! API.                                     |
-| [osu-api-extended](https://github.com/cyperdark/osu-api-extended) | typescript |                                                                                       |
+| [ossapi](https://github.com/circleguard/ossapi)                   | python     | A definitive python wrapper for the osu! api.                                         |
+| [osu-api-extended](https://github.com/cyperdark/osu-api-extended) | typescript | Package for advanced work with "osu" api                                              |
+| [rosu](https://github.com/MaxOhn/rosu)                            | rust       | A rust wrapper for osu!apiv1.                                                         |
```
Comment (jacksonisiah) [inline README.md]: Would it be confusing to combine rosu and v2 into one column somehow? Thoughts?

**Review by jacksonisiah — CHANGES_REQUESTED** [review]
Most of these changes are okay, feel free to add anything else I may miss

File: README.md (hunk 1 above)
Comment (GabuTheDev) [reply README.md:98]: > Not sure why this was changed

Made another heading as title, thought I should also change the sub headings, idk..

File: README.md (hunk 2 above)
Comment (GabuTheDev) [reply README.md]: > Maybe

Maybe.

File: README.md (hunk 4 above)
Comment (GabuTheDev) [reply README.md]: > Would it be confusing to combine rosu and v2 into one column somehow? Thoughts?

I don't think it'll be confusing, but how do you do that? Without changing the whole aspect of the specific list.

File: README.md (hunk 2 above)
Comment (GabuTheDev) [reply README.md]: I got the description from the official wikipedia post about osu!

File: README.md (hunk 4 above)
Comment (jacksonisiah) [inline README.md]: I was thinking something like this:
```suggestion
| [rosu](https://github.com/MaxOhn/rosu) ([api v2](https://github.com/MaxOhn/rosu-v2))                              | rust       | A rust wrapper for osu!api v1/v2.                                                         |
```

File: README.md (hunk 1 above)
Comment (jacksonisiah) [reply README.md:98]: I think it was fine as is

File: README.md (hunk 2 above)
Comment (jacksonisiah) [reply README.md]: I'm more happy with the edit, seems more natural 🤷

File: README.md (hunk 4 above)
Comment (GabuTheDev) [reply README.md]: fixed in https://github.com/CirclePeople/osu-list/pull/9/commits/190e462c1c81909c4e5e99a0a1f153d037c4eb4b

**Review by NiceAesth — APPROVED** [review]
i think this is ok, waiting on @jacksonisiah to re-review

**Review by NiceAesth — APPROVED** [review]
i think this is ok, waiting on @jacksonisiah to re-review

**Review by jacksonisiah — APPROVED** [review]

**Pull request merged**